test/tests/def_blocks.html

```diff
@@ -6,7 +6,8 @@
 <hr>
 
 <blockquote>
-  <p>hello</p>
+  <p>hello
+[2]: hello</p>
```

Comment:
from def_blocks.text:

   > hello
   [2]: hello

I'm not sure this should/should not be parsed as a link definition. It is _outside_ a blockquote.

Reply:
As we now know, this edit was correct because we have rule [A link reference definition cannot interrupt a paragraph.](http://spec.commonmark.org/0.28/#example-178).

Reply:
http://spec.commonmark.org/0.28/#example-210
Ok, because of the laziness rule, '[2]: hello' is part of the blockquote, too, and of the same paragraph of the previous line. So it is not a link def. This is correct then!